Tools/Pipeline/Common/PipelineController.cs

```diff
@@ -481,6 +481,12 @@ private string FindMGCB()
 
         private void DoBuild(string commands)
         {
+            if(!Environment.Is64BitOperatingSystem)
+            {
+                View.OutputAppend("Build failed:  Content can only be compiled from a 64 bit OS.");
```

Comment:
So we do get a message from Windows on 2MGFX and MGCB, but the error message you get is sort of crappy:
```
This version of 2MGFX.exe is not compatible with the version of Windows you're running.  Check your computer's system information and then contact the software publisher.
```
Running from the Pipeline tool we currently get this message:
```
Build failed:  The specified executable is not a valid application for this OS platform.
```
It would be better if all these messages were more specific like `The MonoGame content tools only work on a 64bit OS.`.

To fix this for MGCB.exe or 2MGFX.exe we would have to stop setting the `<ForceArchitecture>x64</ForceArchitecture>` and prefer our own error checking at startup.  Thoughts @KonajuGames  @dellis1972 ?

Even better is us detecting a 32bit OS in the SDK installer and warning the developer at that time.  In fact right now I just fixed the download page to warn the issue.